science/lenstool/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compilers 1.0
+
+name                lenstool
+version             7.1.1
+categories          science
+license             GPL
```

Comment:
We normally specify the license more precisely. Please investigate which version of the GPL they are using. Normally it is something like `GPL-2` or `GPL-2`+ (2 or later).